x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/dependencies.spec.ts

```diff
@@ -43,6 +44,17 @@ describe('Dependencies', () => {
 
       cy.contains('h1', 'postgresql');
     });
+
+    it('has no detectable a11y violations on load', () => {
+      cy.visit(
+        `/app/apm/services/opbeans-java/dependencies?${new URLSearchParams(
+          timeRange
+        )}`
+      );
+      cy.contains('a[role="tab"]', 'Dependencies');
+      // set skipFailures to true to not fail the test when there are accessibility failures
+      checkA11y(true);
```

Comment:
Let's make it clear what `true` means:

```suggestion
      checkA11y({ skipFailures: true });
```